src/Features/Core/Portable/InvertIf/AbstractInvertIfCodeRefactoringProvider.cs

```diff
@@ -51,6 +50,7 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
                 return;
             }
 
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             context.RegisterRefactoring(
                 new MyCodeAction(GetTitle(),
                     c => InvertIfAsync(root, document, ifNode, c)),
```

Comment:
note: i don't know why we're passing the root in here.  could just be removed.

Reply:
That's a previous PR change, will take a look at it.